src/Commands/Server/ToolOperations.cs

```diff
@@ -95,7 +95,7 @@ private async ValueTask<CallToolResponse> OnCallTools(RequestContext<CallToolReq
         var commandContext = new CommandContext(_serviceProvider);
 
         var args = parameters.Params.Arguments != null
-            ? string.Join(" ", parameters.Params.Arguments.Select(kvp => $"--{kvp.Key} \"{kvp.Value}\""))
```

Comment:
CHANGELOG entry? Or what bug is this fixing?

Reply:
It fixed the issue https://github.com/Azure/azure-mcp/issues/307

Reply:
Yeah please click edit on the bug description and include a quick 1/2 sentence of the problem.